sourcecode/apis/contentauthor/database/migrations/2023_10_04_143512_fix_double_encoding_of_library_metadata.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+use App\H5PLibrary;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Database\Migrations\Migration;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        H5PLibrary::whereNotNull('metadata_settings')
```

Comment:
It's a bad idea to use Eloquent models in migrations. As more changes to the database schema are made, the model is changed to reflect the latest version of it, however the migration must work with the latest schema at the time of creation.

To avoid this, and guarantee that future DB or model changes don't break the migration, a raw SQL query should be used instead.